app/views/shelters/show.html.erb

```diff
@@ -6,11 +6,19 @@
   <p>City: <%= @shelter.city %></p>
   <p>State: <%= @shelter.state %></p>
   <p>Zip: <%= @shelter.zip %></p>
-  <p><% @shelter.shelter_reviews.each do  |review| %></p>
-    <p>Title: <%= review.title %></p>
-    <p>Rating: <%= review.rating %></p>
-    <p>Content: <%= review.content %></p>
-    <p>Picture: <%= image_tag review.picture if !review.picture.nil? %></p>
+
+  <h2>Reviews: </h2>
+
+  <p><% @shelter.shelter_reviews.each do |review| %></p>
+    <section id="shelter-review-<%= review.id %>">
+      <p>
+        <strong><%= review.title %></strong>
+        <%= link_to "edit", "/shelters/#{@shelter.id}/#{review.id}/edit" %>
+      </p>
+      <p>Rating: <%= review.rating %></p>
+      <p><%= review.content %></p>
+      <p><%= image_tag review.picture, style: "max-height: 60px; max-width: 60px" if !review.picture.nil? %></p>
+    </section>
```

Comment:
I'm sure we will have merge conflicts here. Each review needs its own section id to be tested properly though. We can change this however we want as long as we keep the id in there.

Reply:
That looks good! We should keep it like that